src/main/java/teammates/common/util/GaeVersionApi.java

```diff
@@ -0,0 +1,73 @@
+package teammates.common.util;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import com.google.appengine.api.modules.ModulesService;
+import com.google.appengine.api.modules.ModulesServiceFactory;
+
+/**
+ * A wrapper class for GAE application admin API. e.g. version management
```

Comment:
No longer accurate, since the function is already narrowed down to version management.